### ISSUE TITLE
Pulmonary artery intimal thickening

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6519]
**Preferred term label:**  Pulmonary artery intimal thickening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Relative intima thickness to the thickness of the vessel wall > 10% (PMID: 28025704)

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary artery morphology HP:0030966

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

Idiopathic pulmonary arterial hypertension ORPHA:275766
Hereditary pulmonary arterial hypertension ORPHA:275777

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  I would revise the define as follows to provode more context for non-specialists
def: Increased thickness of the intimal layer of the pulmonary artery. Throughout the pulmonary arterial tree, the thin intima consists of a non-fenestrated monolayer of endothelial cells lining the vessel lumen, as well as a sub-endothelial interstitium that extends to the internal elastic lamina. Intimal thickness in human lung ranges from 1-16% of total wall thickness. Pulmonary artery intimal thickening can be defined as a relative intima thickness to the thickness of the vessel wall of over 10% (PMID: 28025704 PMID: 23606929

OK?

**Comment by pnrobinson** [comment]
added
